marathon/marathon.go

```diff
@@ -94,6 +95,48 @@ func (m Marathon) ParseApps(jsonBlob []byte) ([]*apps.App, error) {
 	return apps.Apps, err
 }
 
+func (m Marathon) Version() (string, error) {
+	log.WithField("location", m.Location).Debug("asking Marathon for its version")
+	client := m.getClient()
+
+	request, err := http.NewRequest("GET", m.Url("/v2/info"), nil)
+	if err != nil {
+		log.Error(err.Error())
+		return "", err
+	}
+	request.Header.Add("Accept", "application/json")
+
+	infoResponse, err := client.Do(request)
+	if err != nil || (infoResponse.StatusCode != 200) {
+		m.logHTTPError(infoResponse, err)
+		return "", err
+	}
+
+	body, err := ioutil.ReadAll(infoResponse.Body)
+	if err != nil {
+		m.logHTTPError(infoResponse, err)
+		return "", err
+	}
+
+	version, err := m.ParseInfo(body)
+	if err != nil {
+		m.logHTTPError(infoResponse, err)
+		return "", err
+	}
+
+	return version, err
+}
+
+type InfoResponse struct {
+	Version string `json:"version"`
+}
+
+func (m Marathon) ParseInfo(resp []byte) (string, error) {
```

Comment:
since this is only ever returning a version, could you rename it to `ParseVersion`?